packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -2712,6 +2712,29 @@ export class Gloads extends Gload {
   }
 }
 
+// Bth scratch space value of the Ath transaction in the current group
+// Stack: ..., A: uint64, B: uint64 → ..., any
+// Availability: v6
+export class Gloadss extends Gload {
+  /**
+   * Stores scratch space index number according to argument passed.
+   * @param args Expected arguments: [index number]
+   * @param line line number in TEAL file
+   * @param interpreter interpreter object
+   */
+  constructor (args: string[], line: number, interpreter: Interpreter) {
+    // "11" is mock value, will be updated when poping from stack in execute
+    super(["11", "11", ...args], line, interpreter);
```

Comment:
Let's improve the docs and style based on https://github.com/scale-it/algo-builder/pull/608

Reply:
> Let's improve the docs and style based on #608

I update code base @robert-zaremba

Reply:
great! let's fix the test and merge it!